src/scopes/goals/topics.js

```diff
@@ -1,6 +1,7 @@
 import { Op } from 'sequelize';
 import { filterAssociation } from './utils';
 
+/*
```

Comment:
I assume these lines will be uncommented after the new AR, RTR, and topic migration are released. Can we tag them somehow so it is easy to find commented code here (and elsewhere in this PR) that will need to be updated once everything is released?